### PR TITLE
Build: convert addon-docs to use ts-up

### PR DESCRIPTION
**Pull request by ndelangen**
Related: #18732 

## What I did

convert addon-docs to be build using ts-up

### CODE REVIEW

**Comment by ndelangen** [comment]
@tmeasday could you help me out on this one please?

I'm puzzled as to what is causing the e2e tests to fail.. I tried locally, and they passed on my local machine.

**Comment by tmeasday** [comment]
> I also have no idea what's going on with the ci/circleci: chromatic-internal-storybooks task, which also succeeds on my own machine..

Looks like it might have to run on a bigger machine huh? Weird that *removing* stuff from the bundle would cause it to use more memory though.

**Comment by tmeasday** [comment]
@ndelangen I'm running this locally and it's failing the build task:

```
yarn task --task e2e-tests --start-from install --template cra/default-ts --no-link
```

image

**Comment by ndelangen** [comment]
I tried:
> Looks like it might have to run on a bigger machine huh? Weird that removing stuff from the bundle would cause it to use more memory though.

But running in a bigger machine made no difference, I'll try even bigger.

**Comment by ndelangen** [comment]
@tmeasday 
When I run that command:
```
> yarn build-storybook --quiet
✅ install > ✅ compile > ✅ publish > 🔊 run-registry > ✅ sandbox > ✅ build > 🔊 serve > 🔄 e2e-tests


> yarn playwright test

Running 33 tests using 5 workers
  1) [chromium] › addon-controls.spec.ts:62:3 › addon-controls › should apply controls automatically when passed via url 

    page.goto: net::ERR_CONNECTION_RESET at http://localhost:8001/?path=/story/example-button--primary&args=label:Hello+world
    =========================== logs ===========================
    navigating to "http://localhost:8001/?path=/story/example-button--primary&args=label:Hello+world", waiting until "load"
    ============================================================

      61 |
      62 |   test('should apply controls automatically when passed via url', async ({ page }) => {
    > 63 |     await page.goto(`${storybookUrl}?path=/story/example-button--primary&args=label:Hello+world`);
         |                ^
      64 |
      65 |     const sbPage = new SbPage(page);
      66 |     await sbPage.waitUntilLoaded();

        at /Users/me/Projects/Storybook/core/code/e2e-tests/addon-controls.spec.ts:63:16
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -378,7 +378,7 @@ jobs:
             yarn coverage
   chromatic-internal-storybooks:
     executor:
-      class: medium
+      class: medium+
```
Comment (ndelangen) [inline .circleci/config.yml:381]:
```suggestion
      class: medium
```

**Pull request merged**